server/indexer/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Change Log - @hcengineering/model-server-core
+# Change Log - @hcengineering/server-core
```

Comment:
server-indexer? or remove this changelog at all

Reply:
Will on next PR.